kolibri/plugins/registry.py

```diff
@@ -68,7 +68,17 @@ def initialize():
 
         for app in settings.INSTALLED_APPS:
             try:
-                plugin_module = importlib.import_module(app + ".kolibri_plugin")
+
+                # Handle AppConfig INSTALLED_APPS string
+                if ".apps." in app:
+                    # remove .apps.Config line in string
+                    import_string = app.split('.apps.')[0]
+                else:
+                    import_string = app
+
+                import_string += ".kolibri_plugin"
+                plugin_module = importlib.import_module(import_string)
```

Comment:
Oh, right, we missed that AppConfigs can be INSTALLED_APPS when we designed plugin loading: https://docs.djangoproject.com/en/1.11/ref/applications/#configuring-applications